packages/react/src/index.js

```diff
@@ -47,11 +51,19 @@ function Microlink (props) {
   const [state, setState] = useState({})
   const apiUrl = createApiUrl(props)
 
+  const isLazyEnabled = isLazySupported && (lazy === true || isObject(lazy))
+  const lazyOptions = isObject(lazy) ? lazy : {}
```

Comment:
```suggestion
  const lazyOptions = isObject(lazy) ? lazy : undefined
```

I want to be sure we use the `intersectionObserver` default options

Reply:
Good point :+1: